packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -127,37 +125,23 @@ function useMovingAnimation(
 		reset: triggeredAnimation !== finishedAnimation,
 		config: { mass: 5, tension: 2000, friction: 200 },
 		immediate: prefersReducedMotion,
-		onFrame: ( props ) => {
+		onFrame: ( { x, y } ) => {
```

Comment:
Noting that this could be changed to use [method definition syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) (which is a little shorter and names the function).